lib/elixir/lib/exception.ex

```diff
@@ -757,6 +757,19 @@ defmodule KeyError do
   end
 end
 
+defmodule KeyExistsError do
```

Comment:
The name `KeyExistsError` feels "strange" to me. Maybe `DuplicatedKeyError`, but still a little bit strange. IDK 😕